### DISCUSSION TITLE
superset_text.yml

### DISCUSSION THREAD

**Discussion by jikuanyu** [apache/superset#28382] [Q&A / Help]
https://superset.apache.org/docs/installation/pypi

Setting images

    To set the images of your preferred database, admins must create a mapping in the superset_text.yml file with engine and location of the image. The image can be host locally inside your static/file directory or online (e.g. S3)

DB_IMAGES:
  postgresql: "path/to/image/postgres.jpg"
  bigquery: "path/to/s3bucket/bigquery.jpg"
  snowflake: "path/to/image/snowflake.jpg"

but 
I really don't know what I should do when I see this. Where should I put the files? Can anyone clearly tell me？

**Comment by jikuanyu** [comment]
I can't  find superset-frontend

**Reply by mistercrunch** [reply]
https://github.com/apache/superset/tree/master/superset-frontend

**Comment by mistercrunch** [comment]
[superset-frontend/src/assets/images/](https://github.com/apache/superset/tree/master/superset-frontend/src/assets/images/)

**Comment by jikuanyu** [comment]
Thank you for your reply。
  Should I  put superset_text.yml in [superset-frontend/src/assets/images/](https://github.com/apache/superset/tree/master/superset-frontend/src/assets/images/)？I don't have this directory locally 。“superset-frontend”

   I use anaconda3 create env install superset.I install path is ’/opt/yujikuan/anaconda3/envs/superset/lib/python3.11/site-packages/superset/‘ .
I can foud /opt/yujikuan/anaconda3/envs/superset/lib/python3.11/site-packages/superset/static/assets/images. there have some db images.
but I don't know how to take effect?

**Comment by chenlb** [comment]
I has some confuse.

docs on https://superset.apache.org/docs/configuration/databases/#connecting-through-the-ui

but, how to make superset_text.yml take effect in docker compose?
